lib/cocoapods/installer/pod_source_installer.rb

```diff
@@ -133,8 +133,7 @@ def download_source
       def verify_source_is_secure(root_spec)
         return if root_spec.source.nil? || root_spec.source[:http].nil?
         http_source = root_spec.source[:http]
-        return if http_source.downcase.start_with?('https://')
-
+        return if http_source.downcase.start_with?('https://', 'file:///')
```

Comment:
We should parse the URI and check the .scheme property, instead of just doing a string comparison

Reply:
will update

Reply:
updated to use `URI` and check the `scheme`